cpp/arcticdb/CMakeLists.txt

```diff
@@ -800,6 +813,7 @@ if(${TEST})
                 rapidcheck_gtest
                 PRIVATE
                 ${RAPIDCHECK_PRIVATE_LIBRARIES}
+                ${AWSSDK_LINK_LIBRARIES}
```

Comment:
Just noticed this is saying AWS instead of Azure?

Reply:
Hmm it is not added by me. I guess this line is removed in a particular commit, which it should have removed after rebasing.
Removed